implementations/typescript/ockam/ockam_app/src/routes/service/ServiceCreate.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import { invoke } from '@tauri-apps/api/tauri';
+
+  let service = "/service/outlet";
+  let port = "10000";
+
+  async function submit() {
+    await invoke('tcp_outlet_create', {service: service, port: port});
+  }
+</script>
+
+<div>
+  <h1>Service details</h1>
+  <div>
+    <label for="name">Service that you want to share:</label>
+    <div>
+      <input id="name" placeholder="{service}" bind:value="{service}" />
+    </div>
+  </div>
+  <div>
+    <label for="name">Service IP address:</label>
+    <div>
+      <input id="ip_address" placeholder="{port}" bind:value="{port}" />
```

Comment:
Should the element `id` be renamed to port as well?

Reply:
yes! I forgot to rename it